### PR TITLE
SplitChunksPlugin shouldn't change entrypoint chunk name

### PR DESCRIPTION
**Pull request by yurynix**
Hello 👋 

When using the `maxSize` option of `SplitChunksPlugin`, the entry point name is changed to include the hash.
Not sure if that was intended or not.
IMO, it's really important for entrypoints to retain a constant predictable name.

**What kind of change does this PR introduce?**
bugfix

**Did you add tests for your changes?**
yes on a standalone project with multiple entrypoints and a maxSize limit.
A test case is included in this PR. I had to update the snapshot(s) (not 100% sure that's ok)

**Does this PR introduce a breaking change?**
no

**What needs to be documented once your changes are merged?**

### CODE REVIEW

File: test/Defaults.unittest.js [hunk 1]
```diff
@@ -807,7 +807,7 @@ describe("Defaults", () => {
 			-           "idHint": "vendors",
 			-           "priority": -10,
 			-           "reuseExistingChunk": true,
-			-           "test": /[\\\\/]node_modules[\\\\/]/i,
+			-           "test": /[\\\\\\/]node_modules[\\\\\\/]/i,
```
Comment (evilebottnawi) [inline test/Defaults.unittest.js:810]: Remove changes from this tests, bug on old node

**Comment by yurynix** [comment]
https://travis-ci.org/github/webpack/webpack/builds/696989436
Reported ok, but at github it looks hanged 🙁

**Comment by sokra** [comment]
> IMO, it's really important for entrypoints to retain a constant predictable name.

Why is it important? If you need it for inclusion in the HTML, I have to disappoint you. You need to include all splitted files into the HTML. So even the files with a hash.

**Comment by yurynix** [comment]
I don't bundle for the browser (rather for the cloud, i.e. AWS Lambda), I bundle node code with multiple entry points, when I don't use the `maxSize` feature, all good (for me), the output code has exactly the same entry point names as were the input code.

When I _do_ use `maxSize`, a hash is added to the output entry points filenames, which is kinda unexpected, why webpack has this inconsistency? Why in one flow it adds hash but not in the other? 🙂 

Could you elaborate on the benefits of having entry points names with hash?

**Comment by vankop** [comment]
Can not reproduce problem with latest webpack. Please create a new issue with repro if problem still exist.

**Pull request closed without merging**